plugin.video.thinktv/resources/lib/scraper.py

```diff
@@ -0,0 +1,187 @@
+# -*- coding: utf-8 -*-
+# ThinkTV Kodi Video Addon
+#
+from t1mlib import t1mAddon
+import json
+import re
+import os
+import urllib
+import xbmc
+import xbmcplugin
+import xbmcgui
+import HTMLParser
+import sys
+
+h = HTMLParser.HTMLParser()
+qp  = urllib.quote_plus
+uqp = urllib.unquote_plus
+UTF8 = 'utf-8'
+
+
+class myAddon(t1mAddon):
+
+
+  def getShows(self, a, ilist):
+      for b in a:
+          url = b['slug']
+          name = b['title']
+          thumb = b['image']
+          fanart = thumb
+          infoList ={}
+          infoList['Title'] = name
+          infoList['TVShowTitle'] = name
+          plot = b.get('description_long',b.get('description'))
+          infoList['Plot'] = h.unescape(plot)
+          infoList['Genre'] = b.get('genre_titles')
+          infoList['mediatype'] = 'tvshow'
+          c = (name, url, thumb, infoList)
+          contextMenu = [(self.addon.getLocalizedString(30009),'XBMC.RunPlugin(%s?mode=DF&url=AS%s)' % (sys.argv[0], str(c)))]
```

Comment:
please drop XBMC. prefix (was removed already in matrix)

Reply:
fixed